wagtail/api/v2/endpoints.py

```diff
@@ -294,10 +295,7 @@ def get_serializer_class(self):
         request = self.request
 
         # Get model
-        if self.action == 'listing_view':
-            model = self.get_queryset().model
-        else:
-            model = type(self.get_object())
+        model = self.get_queryset().model
```

Comment:
We can't call ``get_object`` when generates API schema. And I think there is no need to call ``get_object`` here anyway.

Reply:
But I think the model of ``get_object`` should be the same with model of queryset, isn't it?